ajax/libs/browser-i18n/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "browser-i18n",
+  "version": "0.0.2",
+  "description": "i18n translation library for client side.",
+  "author": "Renato Gama <renatoargh@gmail.com>",
+  "filename": "i18n.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gammasoft/browser-i18n.git"
+  },
+  "keywords": [
+    "i18n",
+    "browser"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/gammasoft/browser-i18n.git",
+    "files": [
+      "**/*"
```

Comment:
and here we dont need `**/*` all files to auto update,
you can use this `i18n*.js`